test/evergreen/evg_cfg.py

```diff
@@ -30,7 +30,9 @@
 # They are not expected to trigger any 'make check' testing.
 make_check_subdir_skips = [
     "test/csuite",  # csuite has its own set of Evergreen tasks, skip the checking here
-    "test/cppsuite"
+    "test/cppsuite",
+    "test/fuzz",
+    "test/syscall"
```

Comment:
Are fuzz and syscall tests are skipped intentionally here?

Reply:
Yep, they are picked up by the updated `find` command. I added them to the ignore list since we don't perform the `make check directory` task on these test sub-directories. `test/fuzz` isn't something we've put into regression and `test/syscall` is managed by the `syscall-linux` task.

Reply:
Got it, thanks